phpBB/develop/adjust_avatars.php

```diff
@@ -30,6 +30,11 @@
 	die('database not up to date');
 }
 
+if (!isset($config['storage\\avatar\\config\\path']) || $config['storage\\avatar\\config\\path'] != 'phpbb\\storage\\provider\\local')
+{
+	die('use local provider');
```

Comment:
We should display some translateable error message here i think.

Reply:
I dont know, dont know what is the purpose of this folder, but none of the strings in this file are translatable